content/2015.12.hacker-news.rst

```diff
@@ -0,0 +1,176 @@
+Hacker News answers
+###################
+
+:date: 2015-12-15
+:slug: hackernews-feedback
+:lang: en
+:url: hackernews-feedback
+:summary: Where we answer a set of questions that arised on a Hacker News
+          thread.
+
+Today, `Kinto was featured on Hackernews
+<https://news.ycombinator.com/item?id=10733164>`_. We take this as a great
+milestone for the project, even if it also comes with some sort of pressure and
+stress :)
+
+Since a few questions emerged from the thread there, we thought it'd
+appropriate to provide some answers here.
+
+What does this name mean?
+=========================
+
+`Kinto-un <http://dragonball.wikia.com/wiki/Flying_Nimbus>`_ is the name of the
+Flying Nimbus in Dragon ball. It's said that "the cloud can only be ridden by
+someone with a pure heart". Oh, well.
+    
+Maintaining a service is hard
+=============================
+
+The main benefit of Firebase and Parse is that you don't have to run them.
+
+That is true. We want to make it very easy to deploy / use. We can even
+imagine to allow developers to run a Kinto instance in one-click on
+DigitalOcean, Heroku or Bitnami.
+We've started `by making the integration easy on Always Data
+<https://github.com/Natim/auto-kinto-alwaysdata>`.
+
+Or if someone runs a public Kinto instance and wants to charge for its usage,
+that's completely fine!
+
+At Mozilla, we just can't rely on third parties like Firebase or Parse for
+critical data.
+
+If you read french, we've written `an article about our vision on self-hosting
+<http://www.servicedenuages.fr/difficile-promesse-autohebergement>`_
+(yes, we need to translate it! Any help would be welcome on this front.)
+
+Are we using this in production?
+================================
+
+We have different services that are leveraging Kinto:
+
+- The (now dead) reading-list project was using one of the first versions of
+  Kinto
+- We have made `Syncto <https://github.com/mozilla-services/syncto>`_:
+  a service which exposes the API of Kinto and talks with Firefox Sync in the
+  background, so that FirefoxOS developers can use the Kinto.js library
+- We are currently doing the distribution of fonts for Fennec Over The Air
+  using Kinto.
+
+Also, it's not in production yet, but we will use Kinto for the Certificate
+Revocation Lists in the future.
+
+Offline-first support
+=====================
+
+One of the core ideas behind `Kinto.js <https://github.com/Kinto/kinto.js>`_ is
+that it should be offline first: all operations can be done when online or
+offline, and the data is synchronized when the connection goes back to normal.
```

Comment:
Should we add a note about Android and IOS offline first clients? refs https://github.com/Kinto/kinto/issues/327